src/main/java/com/ca/apim/gateway/cagatewayconfig/tasks/zip/builder/BuilderUtils.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2018 CA. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+package com.ca.apim.gateway.cagatewayconfig.tasks.zip.builder;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+import java.util.Map;
+
+class BuilderUtils {
+
+    static Element buildPropertiesElement(final Map<String,Object> properties, final Document document) {
+        Element propertiesElement = document.createElement("l7:Properties");
+        for (String key: properties.keySet()) {
+            Element propertyElement = document.createElement("l7:Property");
+            propertyElement.setAttribute("key", key);
+            Element valueElement;
+            Object value = properties.get(key);
+            if(Integer.class.isAssignableFrom(value.getClass())){
+                valueElement = document.createElement("l7:IntValue");
+            } else if(Integer.class.isAssignableFrom(value.getClass())){
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: This branch can not be reached because the condition duplicates a previous condition in the same sequence of "if/else if" statements ([squid:S1862](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS1862))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=52&issues=AWSQdW1usk71k7lwrENv&open=AWSQdW1usk71k7lwrENv)

[ISSUE_KEY]: AWSQdW1usk71k7lwrENv